flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_FlightPlanPage.js

```diff
@@ -45,6 +45,7 @@ class CDUFlightPlanPage {
             approachRunway = mcdu.flightPlanManager.getApproachRunway();
             if (approachRunway) {
                 destCell += Avionics.Utils.formatRunway(approachRunway.designation);
+                SimVar.SetSimVarValue('L:A32NX_DCDU_APPROACH_RUNWAY_ELEVATION', 'Meters', approachRunway.elevation);
```

Comment:
How is this related to the DCDU ?

Reply:
Nothing really, Can't remember why I called it that. I'll revise

Reply:
Question, why do we need to write to this simvar over and over again with nothing actually changing?

Why don't we change the simvar when we actually change the runway?

Reply:
Sounds good. Can you suggest the file and code block it needs to be moved to?